packages/playground/src/components/smtp_server.vue

```diff
@@ -5,6 +5,9 @@
       <p v-else>Configure your SMTP Server.</p>
     </v-alert>
 
+    <v-alert variant="tonal" type="warning" class="mt-3">
+      SMTP server requires IPv4. Please ensure that your network configuration supports IPv4.
+    </v-alert>
```

Comment:
Since we ensure that IPV4 is enabled if the SMTP is chosen, can we change the phrasing of the alert?

Reply:
- What do u think about this ?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/3d341d60-3480-4590-91e7-16cc6f1ae2c9)